### PR TITLE
CACR-251 Added file to work through code review comments.

### PR DESCRIPTION
**Pull request by juliemtaylor**

### CODE REVIEW

File: new-components/active_exception_policy-comments.html [hunk 1]
```diff
@@ -0,0 +1,798 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+  <link href='http://fonts.googleapis.com/css?family=Open+Sans:400,600,700,800,300' rel='stylesheet' type='text/css'>
+  <link rel="stylesheet" type="text/css" href="https://pro-si.delta.com/etc/designs/deltaPro/homeLib.20170330.css" media="all" />
```
Comment (juliemtaylor) [inline new-components/active_exception_policy-comments.html]: I added in the link to the current CSS file. This helps identify when there are conflicting classes or specificity issues. Which there is one I can see right off. I'll note them below.

File: new-components/active_exception_policy-comments.html [hunk 2]
```diff
@@ -0,0 +1,798 @@
...
+
+
+
+
+
+    <!-- start of new component -->
+
+    <!-- new component styles -->
+<style>
+.v2 .accordion {
```
Comment (juliemtaylor) [inline new-components/active_exception_policy-comments.html]: Because this class exists in the current CSS library there are styles being applied to the the accordion headers that will cause display issues in production. Maybe use the class accordion for the JS and create another class name for the display styling.

File: new-components/active_exception_policy-comments.html [hunk 3]
```diff
@@ -0,0 +1,798 @@
...
+	font-size: 14px;
+	font-weight: 600;
+	font-family: "Open Sans";
+	text-transform: uppercase;
+	color: #333333;
+	text-align: left;
+	padding-left: 10px;
+	padding-top: 5px;
+	width: 164px;
+}
```
Comment (juliemtaylor) [inline new-components/active_exception_policy-comments.html]: Adding `vertical-align: top;` would get this closer to the comp.

File: new-components/active_exception_policy-comments.html [hunk 4]
```diff
@@ -0,0 +1,798 @@
...
+	font-size: 14px;
+	font-weight: 500;
+	font-family: "Open Sans";
+	text-transform: uppercase;
+	color: #87cdff;
+	text-align: right;
+}
+
+.v2 div.panel {
+	background-color: white;
```
Comment (juliemtaylor) [inline new-components/active_exception_policy-comments.html]: Is the html element `div` needed here? This adds a lot of specificity that can be a problem later.

File: new-components/active_exception_policy-comments.html [hunk 5]
```diff
@@ -0,0 +1,798 @@
...
+	color: #0253a4;
+}
+
+.v2 .activebutton {
+	color: #ffffff;
+	background-color: #e3132c;
+	border: 1px solid #e3132c
+}
+
+.v2 .right {
```
Comment (juliemtaylor) [inline new-components/active_exception_policy-comments.html]: It may be better to use the current classes for floats that are in the CSS library.
```
.floatLeft {
	float: left;
}

.floatRight {
	float: right;
}
```

The padding can be added to the container div, which could use a class name.

If you name this div `active-exceptions-header` and then rename the current `active-exceptions-header` to `active-exceptions-panel` and the `panel` to `active-exceptions-panel-open` or `active-exceptions-panel-active`, that may make more sense semantically.

That way you can reserve the class `accordion` for just the JS action.

File: new-components/active_exception_policy-comments.html [hunk 6]
```diff
@@ -0,0 +1,798 @@
...
+
+.v2 .showtable {
+	display: block;
+	border-bottom: none;
+	margin-left: -5px;
+	border: 1px solid #c7c7c7;
+	width: 958px;
+}
+
+.v2 .hidetable {
```
Comment (juliemtaylor) [inline new-components/active_exception_policy-comments.html]: There is a class in the current library that can be used instead.
```
.noDisplay {
	display: none;
}
```

File: new-components/active_exception_policy-comments.html [hunk 7]
```diff
@@ -0,0 +1,798 @@
...
+	display: none;
+}
+
+.v2 .viewdetails {
```
Comment (juliemtaylor) [inline new-components/active_exception_policy-comments.html]: This class may work better as `active-exception-header-link`.

File: new-components/active_exception_policy-comments.html [hunk 8]
```diff
@@ -0,0 +1,798 @@
...
+	border: 1px solid #e3132c
+}
+
+.v2 .right {
+	float: right;
+	padding-right: 10px;
+	padding-top: 10px;
+}
+
+.v2 .left {
```
Comment (juliemtaylor) [inline new-components/active_exception_policy-comments.html]: Same comment as line 155.

File: new-components/active_exception_policy-comments.html [hunk 9]
```diff
@@ -0,0 +1,798 @@
...
+	float: left;
+	padding-left: 10px;
+	padding-top: 10px;
+}
+
+.v2 .viewdetails:hover {
+	color: #a9dbff;
+}
+
+.v2 .inner table {
```
Comment (juliemtaylor) [inline new-components/active_exception_policy-comments.html]: Can you add a class name to this table and apply the margin that way?

Additionally, the class `inner` may work better as something like `active-exception-panel-inner`.

File: new-components/active_exception_policy-comments.html [hunk 10]
```diff
@@ -0,0 +1,798 @@
...
+	margin-top: 20px;
+	margin-bottom: 0;
+	margin-left: 17px;
+}
+
+.v2 .inner .active-exception-content {
+	margin-left: 0px;
+}
+
+.v2 .inner .showtable {
```
Comment (juliemtaylor) [inline new-components/active_exception_policy-comments.html]: If the border is put on a div wrapper (ref comments from line 111), this class isn't needed.

File: new-components/active_exception_policy-comments.html [hunk 11]
```diff
@@ -0,0 +1,798 @@
...
+	border: none;
+}
+
+.v2 .theading {
```
Comment (juliemtaylor) [inline new-components/active_exception_policy-comments.html]: Maybe this class name would work better as `active-exceptions-table-head`.

File: new-components/active_exception_policy-comments.html [hunk 12]
```diff
@@ -0,0 +1,798 @@
...
+<style>
+.v2 .accordion {
+	height: 42px;
+	display: inline-block;
+	background-color: #02132d;
+	color: #ffffff;
+	width: 100%;
+	font-size: 18px;
+	font-weight: 500;
+	font-family: "Open Sans";
```
Comment (juliemtaylor) [inline new-components/active_exception_policy-comments.html]: Since the current CSS library has Open Sans set as the body font, it doesn't need to be declared here.

File: new-components/active_exception_policy-comments.html [hunk 13]
```diff
@@ -0,0 +1,798 @@
...
+}
+
+.v2 .hidetable {
+	display: none;
+}
+
+.v2 .viewdetails {
+	font-size: 14px;
+	font-weight: 500;
+	font-family: "Open Sans";
```
Comment (juliemtaylor) [inline new-components/active_exception_policy-comments.html]: Open Sans is set as the body font, it doesn't need to be declared here.

File: new-components/active_exception_policy-comments.html [hunk 14]
```diff
@@ -0,0 +1,798 @@
...
+	border: 1px solid #c7c7c7;
+	margin-left: 40px;
+	border-top: none;
+	width: 958px;
+	margin-bottom: 20px;
+}
+
+.v2 .active-exception-content table tr:nth-child(4n), .v2 .active-exception-content table tr:nth-child(4n-1)
+	{
+	background: #ffffff;
```
Comment (juliemtaylor) [inline new-components/active_exception_policy-comments.html]: I would use `background-color: #fff;` here.

File: new-components/active_exception_policy-comments.html [hunk 15]
```diff
@@ -0,0 +1,798 @@
...
+.v2 .marginbottom {
+	margin-bottom: 0;
+}
+
+.v2 .asteriskcopy {
+	display: none;
+	height: 42px;
+	font-size: 14px;
+	font-weight: 500;
+	font-family: 'Open Sans';
```
Comment (juliemtaylor) [inline new-components/active_exception_policy-comments.html]: Open Sans is set as the body font, it doesn't need to be declared here.

File: new-components/active_exception_policy-comments.html [hunk 16]
```diff
@@ -0,0 +1,798 @@
...
+	padding-left: 10px;
+	padding-top: 10px;
+	margin-left: -5px;
+}
+
+.v2 .activediv, .v2 .pastdiv {
+	float: left;
+	font-size: 20px;
+	font-weight: 500;
+	font-family: 'Open Sans';
```
Comment (juliemtaylor) [inline new-components/active_exception_policy-comments.html]: Open Sans is set as the body font, it doesn't need to be declared here.

File: new-components/active_exception_policy-comments.html [hunk 17]
```diff
@@ -0,0 +1,798 @@
...
+	text-transform: uppercase;
+	color: #333333;
+	text-align: left;
+	padding-left: 10px;
+	padding-top: 5px;
+	width: 164px;
+}
+
+.v2 .tdata {
+	font-family: "Open Sans";
```
Comment (juliemtaylor) [inline new-components/active_exception_policy-comments.html]: Open Sans is set as the body font, it doesn't need to be declared here.

File: new-components/active_exception_policy-comments.html [hunk 18]
```diff
@@ -0,0 +1,798 @@
...
+	list-style: disc;
+	padding-left: 16px;
+	font-weight: 700;
+	text-transform: none;
+	font-size: 16px;
+	color: #0b1f66;
+}
+
+.v2 .timetext {
+	font-family: 'Open Sans';
```
Comment (juliemtaylor) [inline new-components/active_exception_policy-comments.html]: Open Sans is set as the body font, it doesn't need to be declared here.

File: new-components/active_exception_policy-comments.html [hunk 19]
```diff
@@ -0,0 +1,798 @@
...
+	text-align: center;
+	width: 136px;
+}
+
+.v2 #viewmore:hover {
+	background-color: #cb1127;
+}
+
+.v2 .refundrequest {
+	font-family: 'Open Sans';
```
Comment (juliemtaylor) [inline new-components/active_exception_policy-comments.html]: Open Sans is set as the body font, it doesn't need to be declared here.

File: new-components/active_exception_policy-comments.html [hunk 20]
```diff
@@ -0,0 +1,798 @@
...
+.v2 .osifieldblock {
+	height: auto;
+	background-color: #22659c;
+	padding: 10px;
+}
+
+.v2 .fieldheader {
+	font-size: 20px;
+	font-weight: 600;
+	font-family: "Open Sans";
```
Comment (juliemtaylor) [inline new-components/active_exception_policy-comments.html]: Open Sans is set as the body font, it doesn't need to be declared here.

File: new-components/active_exception_policy-comments.html [hunk 21]
```diff
@@ -0,0 +1,798 @@
...
+	text-align: center;
+	text-transform: uppercase;
+	color: #ffffff;
+}
+
+.v2 .fieldcontent {
+	font-size: 14px;
+	font-weight: 500;
+	font-family: "Open Sans";
```
Comment (juliemtaylor) [inline new-components/active_exception_policy-comments.html]: Open Sans is set as the body font, it doesn't need to be declared here.

File: new-components/active_exception_policy-comments.html [hunk 22]
```diff
@@ -0,0 +1,798 @@
...
+	text-align: center;
+	text-transform: none;
+	color: #87cdff;
+}
+
+.v2 .osifield {
+	font-size: 16px;
+	font-weight: 600;
+	font-family: "Open Sans";
```
Comment (juliemtaylor) [inline new-components/active_exception_policy-comments.html]: Open Sans is set as the body font, it doesn't need to be declared here.

File: new-components/active_exception_policy-comments.html [hunk 23]
```diff
@@ -0,0 +1,798 @@
...
+.v2 .fieldcontent {
+	font-size: 14px;
+	font-weight: 500;
+	font-family: "Open Sans";
+	text-align: center;
+	text-transform: none;
+	color: #87cdff;
+}
+
+.v2 .osifield {
```
Comment (juliemtaylor) [inline new-components/active_exception_policy-comments.html]: What does this class name refer to?

File: new-components/active_exception_policy-comments.html [hunk 24]
```diff
@@ -0,0 +1,798 @@
...
+	text-transform: uppercase;
+	font-size: 20px;
+	color: #0b1f66;
+	font-weight: 700;
+	text-align: left;
+	width: 254px;
+	padding-left: 10px;
+}
+
+.v2 ul.ulist {
```
Comment (juliemtaylor) [inline new-components/active_exception_policy-comments.html]: Is the html element `ul` needed here? This adds additional specificity that can be a problem later. Additionally, lets reevaluate this class name.

File: new-components/active_exception_policy-comments.html [hunk 25]
```diff
@@ -0,0 +1,798 @@
...
+	list-style: disc;
+	padding-left: 16px;
+	font-weight: 700;
+	text-transform: none;
+	font-size: 16px;
+	color: #0b1f66;
+}
+
+.v2 .timetext {
```
Comment (juliemtaylor) [inline new-components/active_exception_policy-comments.html]: Let's talk about the class names for all these table cell elements.

File: new-components/active_exception_policy-comments.html (hunk 4 above)
Comment (PradeepaJayaprakash) [inline new-components/active_exception_policy-comments.html]: This is fixed.

File: new-components/active_exception_policy-comments.html (hunk 12 above)
Comment (PradeepaJayaprakash) [inline new-components/active_exception_policy-comments.html]: This is fixed.

File: new-components/active_exception_policy-comments.html (hunk 6 above)
Comment (PradeepaJayaprakash) [inline new-components/active_exception_policy-comments.html]: This is fixed.

File: new-components/active_exception_policy-comments.html (hunk 13 above)
Comment (PradeepaJayaprakash) [inline new-components/active_exception_policy-comments.html]: This is fixed.

File: new-components/active_exception_policy-comments.html (hunk 7 above)
Comment (PradeepaJayaprakash) [inline new-components/active_exception_policy-comments.html]: This is done

File: new-components/active_exception_policy-comments.html (hunk 14 above)
Comment (PradeepaJayaprakash) [inline new-components/active_exception_policy-comments.html]: This is done.

File: new-components/active_exception_policy-comments.html [hunk 26]
```diff
@@ -0,0 +1,798 @@
...
+	width: 958px;
+	margin-bottom: 20px;
+}
+
+.v2 .active-exception-content table tr:nth-child(4n), .v2 .active-exception-content table tr:nth-child(4n-1)
+	{
+	background: #ffffff;
+}
+
+.v2 .active-exception-content table tr:nth-child(4n-2), .v2 .active-exception-content table tr:nth-child(4n-3)
```
Comment (PradeepaJayaprakash) [inline new-components/active_exception_policy-comments.html]: This is done

File: new-components/active_exception_policy-comments.html [hunk 27]
```diff
@@ -0,0 +1,798 @@
...
+
+.v2 .borderbottom {
+	border-bottom: none;
+}
+
+.v2 .marginbottom {
+	margin-bottom: 0;
+}
+
+.v2 .asteriskcopy {
```
Comment (PradeepaJayaprakash) [inline new-components/active_exception_policy-comments.html]: Changed it to active-exceptions-disclaimer

File: new-components/active_exception_policy-comments.html (hunk 15 above)
Comment (PradeepaJayaprakash) [inline new-components/active_exception_policy-comments.html]: This is done

File: new-components/active_exception_policy-comments.html (hunk 16 above)
Comment (PradeepaJayaprakash) [inline new-components/active_exception_policy-comments.html]: This is done.

File: new-components/active_exception_policy-comments.html (hunk 11 above)
Comment (PradeepaJayaprakash) [inline new-components/active_exception_policy-comments.html]: Done

File: new-components/active_exception_policy-comments.html (hunk 17 above)
Comment (PradeepaJayaprakash) [inline new-components/active_exception_policy-comments.html]: done

File: new-components/active_exception_policy-comments.html (hunk 24 above)
Comment (PradeepaJayaprakash) [inline new-components/active_exception_policy-comments.html]: done

File: new-components/active_exception_policy-comments.html (hunk 18 above)
Comment (PradeepaJayaprakash) [inline new-components/active_exception_policy-comments.html]: done

File: new-components/active_exception_policy-comments.html (hunk 19 above)
Comment (PradeepaJayaprakash) [inline new-components/active_exception_policy-comments.html]: done

File: new-components/active_exception_policy-comments.html (hunk 20 above)
Comment (PradeepaJayaprakash) [inline new-components/active_exception_policy-comments.html]: done

File: new-components/active_exception_policy-comments.html (hunk 21 above)
Comment (PradeepaJayaprakash) [inline new-components/active_exception_policy-comments.html]: done

File: new-components/active_exception_policy-comments.html (hunk 22 above)
Comment (PradeepaJayaprakash) [inline new-components/active_exception_policy-comments.html]: done

File: new-components/active_exception_policy-comments.html (hunk 1 above)
Comment (PradeepaJayaprakash) [inline new-components/active_exception_policy-comments.html]: Is this regarding the background color ? I guess once you put the component on the actual page, this will get fixed. But as you said, I can see an issue with accordion class.

File: new-components/active_exception_policy-comments.html (hunk 2 above)
Comment (PradeepaJayaprakash) [inline new-components/active_exception_policy-comments.html]: Will change the class name.

Comment (PradeepaJayaprakash) [inline new-components/active_exception_policy-comments.html]: Changed it to exception-accordion

File: new-components/active_exception_policy-comments.html [hunk 28]
```diff
@@ -0,0 +1,798 @@
...
+	margin-bottom: 30px;
+}
+
+.v2 .exception-policy-content {
+	width: 960px;
+	margin-left: auto;
+	margin-right: auto;
+}
+
+.v2 .osihr{
```
Comment (PradeepaJayaprakash) [inline new-components/active_exception_policy-comments.html]: Changed it to exception-osi-hr

File: new-components/active_exception_policy-comments.html (hunk 23 above)
Comment (PradeepaJayaprakash) [inline new-components/active_exception_policy-comments.html]: Changed it to exception-osi-field-content

File: new-components/active_exception_policy-comments.html (hunk 5 above)
Comment (PradeepaJayaprakash) [inline new-components/active_exception_policy-comments.html]: Changed panel to exceptions-panel and inner to exceptions-panel-inner. I am not adding active to class names as same code will be reused for past exceptions too. So keeping it general.
